### PR TITLE
some mappings

### PR DESCRIPTION
**Pull request by marcin212**

### CODE REVIEW

File: mappings/net/minecraft/sortme/NativeImage.mapping [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		ARG 3 height
+	METHOD a getWidth ()I
+	METHOD a getPixelRGBA (II)I
+		ARG 1 x
+		ARG 2 y
+	METHOD a setPixelRGBA (III)V
+		ARG 1 x
+		ARG 2 y
+	METHOD a resizeSubRectTo (IIIILcki;)V
+	METHOD a readFromInputStream (Lcki$a;Ljava/io/InputStream;)Lcki;
```
Comment (asiekierka) [inline mappings/net/minecraft/sortme/NativeImage.mapping]: fromInputStream/fromByteBuffer

File: mappings/net/minecraft/sortme/NativeImage.mapping [hunk 2]
```diff
@@ -0,0 +1,28 @@
+CLASS cki net/minecraft/sortme/NativeImage
```
Comment (asiekierka) [inline mappings/net/minecraft/sortme/NativeImage.mapping]: client/texture or client/util, maybe?

File: mappings/net/minecraft/block/BlockSugarCane.mapping [hunk 3]
```diff
@@ -1,4 +1,5 @@
 CLASS bke net/minecraft/block/BlockSugarCane
+	FIELD a age Lboo;
```
Comment (asiekierka) [inline mappings/net/minecraft/block/BlockSugarCane.mapping]: I'll have an automated mechanism for this ,but if you want to keep it, static final fields are generally UPPERCASE_WITH_SNAKES.

File: mappings/net/minecraft/advancement/AdvancementFrame.mapping [hunk 4]
```diff
@@ -8,6 +8,7 @@ CLASS v net/minecraft/advancement/AdvancementFrame
 	METHOD <init> (Ljava/lang/String;ILjava/lang/String;ILa;)V
 		ARG 3 id
 		ARG 4 texV
+	METHOD a getID ()Ljava/lang/String;
```
Comment (asiekierka) [inline mappings/net/minecraft/advancement/AdvancementFrame.mapping]: getId, always camelCase with no extra large letters.

**Review by asiekierka — CHANGES_REQUESTED** [review]

**Review by modmuss50 — APPROVED** [review]
All looks fine to me

**Pull request merged**